HW#13/From molecules to atoms.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+
+def from_molecules_to_atoms(formula):
```

Comment:
There are different types of brackets: `{`,`(``[` possible, so it’s better to unify them: `formula.replace('[', '(')` or `formula.replace('{', '(')`. - to one type